### DISCUSSION TITLE
undocumented-variations

### DISCUSSION THREAD

**Discussion by wenjiany** [PharmGKB/PharmCAT#183] [General]
v2.12, Based on https://pharmcat.org/methods/NamedAlleleMatcher-201/#undocumented-variations, "A different approach is taken for genes for which the defined variants affect drug toxicity. When these "undefined" variant calls are encountered, they will be treated as reference."
Can some rationale be added to support this decision? It might be helpful to indicate such situation clearly in the html report.

**Comment by BinglanLi** [comment]
PharmCAT provides drug-prescribing recommendations based on the guidelines published by expert groups like CPIC, DPWG, etc. 

The recommendations are generally provided for phenotypes that are derived based on an individual's genotypes. In the guidelines, the expert groups provide allele definitions (genomic positions and specific nucleotide changes). PharmCAT uses these allele definitions to identify the genotype per gene from the VCF input file. 

By default, only genetic variations that are defined in the allele definitions can be mapped to genotypes by the Named Allele Matcher. If the sample includes a variant that is not included in the allele definitions at an allele-defining position, the `Named Allele Matcher` produces a ‘Not called’ for the affected gene since the sample matches neither the reference nor any defined variant.

A “Not called” output cannot be connected to a guideline recommendation, even if the sample has other actionable variants. In the interest of providing recommendation guidance, the decision was made to set the undocumented variants to reference for the genes listed on the PharmCAT webpage (https://pharmcat.org/methods/NamedAlleleMatcher-201/). 

PharmCAT reports the undocumented variation in Section III: Allele Matching Details of the report.

Please feel free to re-open the issue if you have further questions.
